src/Makefile.am

```diff
@@ -66,7 +66,7 @@ if BUILD_BITCOIND
 endif
 
 if BUILD_BITCOIN_UTILS
-  bin_PROGRAMS += bitcoin-cli bitcoin-tx
+  bin_PROGRAMS += bitcoin-cli bitcoin-tx bitcoin-fuzzy
```

Comment:
I don't think this should be under BITCOIN_UTILS, as that means it will be packaged as part of the binaries.
It needs to be in its own category, to be specifically enabled as a configure switch, I think.